explorer/src/__tests__/realtime.test.ts

```diff
@@ -182,4 +182,24 @@ describe('realtime', () => {
       },
     )
   })
+
+  it('rejects multiple connections from single node', async done => {
+    expect.assertions(4)
+
+    let ws1: WebSocket
+    let ws2: WebSocket
+
+    const onCloseCallback = (event: WebSocket.CloseEvent) => {
+      expect(ws1.readyState).toBe(3) // connection closed
```

Comment:
How do you feel about using the [constants](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket#Constants) for these?